Assignment/Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿while (true) {
+   Console.Write ("Give number of rows : ");
+   if (int.TryParse (Console.ReadLine (), out int n) && (n > 0)) {
+      int i, l, s, j;
+      for (i = -n; i <= n; i++) {
+         //condition to change negative value of i to positive using l variable to calculate space
+         if (i < 0)
+            l = -i;
+         else
+            l = i;
+         for (s = 0; s < l; s++)
+            Console.Write ("  ");
+         for (j = 0; j < 2 * (n - l) + 1; j++)
+            Console.Write ("* ");
+         Console.WriteLine ();
       }
-   }
+      break;

```

Comment:
Why we need infinite loop for this program.

Reply:
To get valid input from the user for that only I'm using an infinite while loop.